src/client.cpp

```diff
@@ -736,7 +736,7 @@ Status Client::Impl::uploadObjectStream(id_t accountId, id_t instrumentId,
         return makeNetworkError();
     }
 
-    if (cs->getResponseCode() != 201)
+    if (cs->getResponseCode() != 201 && cs->getResponseCode() != 200)
```

Comment:
But 2xx code may not correspond to operation. 200 isn't appropriate for resource creation (post), but 201 is.